### ISSUE TITLE
Current Recall at 0 during actual training? 

### ISSUE THREAD

**Issue by Moondra** [experiencor/basic-yolo-keras#209]
First I pretrained the model for a couple of epochs.
During pretraining the recall starts off around 50 or so and drops steadily (which seems to be normal).

However, when I start regular training, the recall is consistently [0]. Not sure what I could be doing wrong.
Here is my json file:

   

    {
        "model" : {
            "architecture":         "Inception3",
            "input_size":           299,
            "anchors":              [2.95,6.42, 4.53,2.38, 6.91,3.16, 7.93,4.66, 8.46,7.42],
            "max_box_per_image":    12,        
            "labels":               ["fish"]
        },

        "train": {
            "train_image_folder":   "C:/Users/Moondra/Desktop/ object Detection/train_image_folder/",
            "train_annot_folder":   "C:/Users/Moondra/Desktop/ object Detection/train_annot_folder/",     
              
            "train_times":          30,
            "pretrained_weights":   "new_weights.h5",
            "batch_size":           20,
            "learning_rate":        1e-4,
            "nb_epoch":             15,
            "warmup_epochs":        0,

            "object_scale":         4.0,
            "no_object_scale":      1.0,
            "coord_scale":          2.0,
            "class_scale":          1.0,

            "saved_weights_name":   "new_weights.h5",
            "debug":                true
        },

        "valid": {
            "valid_image_folder":   "C:/Users/Moondra/Desktop/object Detection/valid_image_folder/",
            "valid_annot_folder":   "C:/Users/Moondra/Desktop/object Detection/valid_annot_folder/",

            "valid_times":          1
        }
    }

**Comment by Moondra** [comment]
Actually, after one epoch it seems to be increasing (non-zero numbers). I guess I need to train for a longer period of time.
